### ISSUE TITLE
False positive no-duplicate-imports for flow types

### ISSUE THREAD

**Issue by ayrton** [eslint/eslint#6214]
**What version of ESLint are you using?**

2.8.0

**What parser (default, Babel-ESLint, etc.) are you using?**

babel-eslint

**What did you do? Please include the actual source code causing the issue.**

```js
import { trackClick } from 'analytics';
import type { WithUrl } from 'analytics';
```

**What did you expect to happen?**

No errors, because as far as I know there's no way to merge both imports.

**What actually happened? Please include the actual, raw output from ESLint.**

```
error  'analytics' import is duplicated  no-duplicate-imports`
```

**Comment by nzakas** [comment]
Unfortunately, I don't think we can do anything about this. We don't natively support Flow, so we can't put in exceptions for it. Your best bet is to either manually disable the rule or file an issue with babel-eslint to see if maybe they can do something on that end.

**Comment by mysticatea** [comment]
I think [this issue](https://github.com/babel/eslint-plugin-babel/issues/59) is for this.

**Comment by nzakas** [comment]
Thanks @mysticatea. That seems like the right place to continue discussing.

**Comment by ayrton** [comment]
You're right, thanks
